web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -0,0 +1,51 @@
+# VM details - boot order
+
+## Entering edit mode
+![VM details - Overview tab](img/C1-0-0.jpg)
```

Comment:
nit: we are now showing underlines for each of the section headers instead of the (?) icon.